### DISCUSSION TITLE
Welcome to crossplane Discussions!

### DISCUSSION THREAD

**Discussion by negz** [crossplane/crossplane#2171] [General]
## 👋 Welcome!

The Crossplane community is experimenting with GitHub Discussions. We encourage you to use discussions to:

* Ask questions about Crossplane.
* Get help with Crossplane.
* Share tips for using Crossplane.
* Share solutions to common gotchas.

Please refrain from using discussions to report bugs and to propose features or enhancements - we encourage you to use issues for these. Note that discussions are only enabled on `crossplane/crossplane` for the time being. We encourage you to keep raising issues near the relevant code (e.g. raise AWS specific issues on [`crossplane/provider-aws`](https://github.com/crossplane/provider-aws)), but keep all discussion here. 🙂

**Comment by samirparhi-dev** [comment]
Hello Together,

I am a newbie to this crossplane, as I understood this is a warper on top of K8s for multi-cloud. is my understanding correct?

**Reply by muvaf** [reply]
Hi @samirparhi-dev ! It's an addon to Kubernetes that allows you provision & orchestrate cloud resources in a Kubernetes-native way. The docs here would be helpful to get started: https://crossplane.io/docs/v1.0/

**Comment by danielpsf** [comment]
Hi all,

I'm looking for a tool exactly like Crossplane or Kubevela, in fact, I'm working on building a platform on a multi-cloud setup that mainly consists of K8S and some services from AWS, Azure, and potentially, in the future, Alicloud.

I was just curious regarding some topics, like:

- Any thoughts regarding how Crossplane will tackle some of the Serverless topics in the future (Knative/Kubeless/OpenFaas/OpenWhisk/etc)?
- Do you provide a way for us to write custom CRDs? I mean, if we ever want to write something for Instana for example, or something like that, but that is purely tightened to our context and we can't contribute upstream.
- Any tips for us starting to build our platform on top of Crossplane?

BTW, found you guys through https://www.infoq.com/articles/crossplane-paas-kubernetes/. ;) 

Cheers,
Daniel.

**Reply by danielpsf** [reply]
For the Serverless stuff, I found this on the docs: https://crossplane.io/docs/v0.11/guides/guides.html... Sorry I didn't dig it up first! 😉

**Reply by muvaf** [reply]
Hi @danielpsf !

> Do you provide a way for us to write custom CRDs?

@danielpsf yes, you can develop your own provider using the [template](https://github.com/crossplane/provider-template) and the [guide](https://crossplane.io/docs/v1.1/contributing/provider_development_guide.html).

> Any tips for us starting to build our platform on top of Crossplane?

I think starting with Composition would be the best place to start.
